release-notes/opensearch-release-notes-1.3.8.md

```diff
@@ -0,0 +1,34 @@
+# OpenSearch and OpenSearch Dashboards 1.3.8 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 1.3.8 includes the following bug fixes, infrastructure and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.8.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.8.md).
+
+## ENHANCEMENTS
+
+### OpenSearch Security
+* Username validation for special characters ([#2277](https://github.com/opensearch-project/security/pull/2277))
+* Tool scripts are updated to run on Windows ([#2371](https://github.com/opensearch-project/security/pull/2371))
+* Exclude the term + .keyword when excluding fields ([2375](https://github.com/opensearch-project/security/pull/2375))
+
+## BUG FIXES
+
+### OpenSearch Anomaly Detection
+* Fixed DLS/FLS logic around numeric aggregations ([#789](https://github.com/opensearch-project/anomaly-detection/pull/789))
+
+## INFRASTRUCTURE
+
+### OpenSearch Anomaly Detection
+* Added windows and mac CI to 1.3 ([#745](https://github.com/opensearch-project/anomaly-detection/pull/745))
+
+## MAINTENANCE
+
+### OpenSearch Security
+* Update cxf-core to 3.5.5 ([#2349](https://github.com/opensearch-project/security/pull/2349))
+
+### OpenSearch Dashboards Security
```

Comment:
Could you make sure this naming is align with our convention? For example, https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.5.0.md#opensearch-security-dashboards-plugin

Reply:
Sure Zelin